extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/StaticResourceConfig.java

```diff
@@ -0,0 +1,59 @@
+package io.quarkus.vertx.http.runtime;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.OptionalInt;
+
+import io.quarkus.runtime.annotations.ConfigDocMapKey;
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class StaticResourceConfig {
```

Comment:
@gsmet this is the configuration group evoked in https://github.com/quarkusio/quarkus/pull/4363/files#r483076835